blueprint/index.md

```diff
@@ -0,0 +1,169 @@
+---
+title: Develop a React App with Typescript that uses the Genesys Cloud Platform SDK
+author: Jacob Shaw
+indextype: blueprint
+icon: blueprint
+image: images/5.png
+category: 6
+summary: |
+  This Genesys Cloud Developer Blueprint demonstrates how to integrate the Genesys Cloud Javascript Platform SDK into your React app.  This example uses Typescript, although the implementation will be similar if you're using plain javascript.  The sample app provides examples of use cases such as creating a profile page for a user and admin operations like signing all the users out of a queue.  The blueprint will describe the steps needed to integrate the Genesys Cloud SDK into your React app.
+---
+
+<!-- no toc -->
```

Comment:
What's this no toc comment for? 

If you do want an auto-generated TOC, use `[toc]`. That's fully up to you though.